TokenFactory2/app/js/index.js

```diff
@@ -17,13 +17,11 @@ $(document).ready(function() {
       });
   });
 
-  $("#useToken button").click(function() {
+  $('#useToken button').click(function() {
     var address = $('#useToken input').val();
     console.dir(Token.abi);
-    currentToken = new EmbarkJS.Contract({
-      abi: Token.options.jsonInterface,
-      address: address
-    });
+    console.log(Token._jsonInterface);
+    currentToken = new web3.eth.Contract(Token._jsonInterface, address);
```

Comment:
`Token.options.jsonInterface` should still be working.
Also, `new EmbarkJS.Contract` probably just needs `web3: web3`